src/main/kotlin/uk/gov/justice/digital/assessments/api/AssessmentDto.kt

```diff
@@ -2,13 +2,11 @@ package uk.gov.justice.digital.assessments.api
 
 import io.swagger.v3.oas.annotations.media.Schema
 import uk.gov.justice.digital.assessments.jpa.entities.assessments.AssessmentEntity
+import uk.gov.justice.digital.assessments.jpa.entities.assessments.SubjectEntity
 import java.time.LocalDateTime
 import java.util.UUID
 
-class AssessmentDto(
-
-  @Schema(description = "Assessment primary key", example = "1234")
```

Comment:
removed the internal IDs from the exposed dtos as we don't want to exposed numeric subsequent ids for security